content/v2.0/influxdb-templates/use.md

```diff
@@ -1,20 +1,193 @@
 ---
-title: Use an InfluxDB template
+title: Use InfluxDB templates
 description: >
-  placeholder
+  Use the `influx pkg` command to view and install templates from your local
+  filesystem or from URLs.
 menu:
   v2_0:
     parent: InfluxDB templates
-    name: Use a template
+    name: Use templates
 weight: 101
 v2.0/tags: [templates]
 ---
 
-## Placeholder outline
-- Use a template
-  - `influx pkg`
-  - Populating secrets for templates that include secrets
-  - View package summaries before installing
-  - Validate a package before installing
-  - Use a template from a remote URL  
-  - Define secrets while installing a template
+Use the `influx pkg` command to summarize, validate, and install templates from
+your local filesystem and from URLs.
+
+- [View a template summary](#view-a-template-summary)
+- [Validate a template](#validate-a-template)
+- [Install templates](#install-templates)
+- [Use InfluxDB community templates](#use-influxdb-community-templates)
+
+## View a template summary
+To view a summary of what's included in a template before installing the template,
+use the [`influx pkg summary` command](/v2.0/reference/cli/influx/pkg/summary/).
+Summarize templates stored in your local filesystem or from a URL.
+
+{{% code-tabs-wrapper %}}
+{{% code-tabs %}}
+[From a file](#)
+[From a URL](#)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+```sh
+# Syntax
+influx pkg summary -f <template-file-path>
+
+# Example
+influx pkg summary -f /path/to/template.yml
+```
+{{% /code-tab-content %}}
+{{% code-tab-content %}}
+```sh
+# Syntax
+influx pkg summary -u <template-url>
+
+# Example
+influx pkg summary -u https://raw.githubusercontent.com/influxdata/community-templates/master/linux_system/linux_system.yml
+```
+{{% /code-tab-content %}}
+{{% /code-tabs-wrapper %}}
+
+## Validate a template
+To validate a template before installing it or for troubleshooting purposes, use
```

Comment:
To validate a template before you install it or troubleshoot an existing template